### PR TITLE
fix: combobox height issues

### PR DESCRIPTION
**Pull request by kylemcd**
### Description 
- Fixes height issues seen with the combobox trigger
- Adds min height and min width as prop values for layout components
- Fixes incorrect reference to flex wrap in Stack and adds shorthand

### Tasks
[KNO-6406](https://linear.app/knock/issue/KNO-6406/[telegraph]-fix-height-issue-with-combobox)

### CODE REVIEW

**Comment by kylemcd** [comment]
* **#185** <a href="https://app.graphite.dev/github/pr/knocklabs/telegraph/185?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kylemcd and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/combobox/src/Combobox/Combobox.tsx [hunk 1]
```diff
@@ -202,19 +194,16 @@ const TriggerValue = <T extends TgphElement>({
...
     if (contextValue.length === 0) {
       return (
-        <Stack h={height} align="center" my="1">
-          <TelegraphButton.Text color="gray">
-            {context.placeholder}
-          </TelegraphButton.Text>
-        </Stack>
+        <TelegraphButton.Text color="gray">
+          {context.placeholder}
+        </TelegraphButton.Text>
```
Comment (kylemcd) [inline packages/combobox/src/Combobox/Combobox.tsx:199]: The height management for `TriggerValue` has been moved to `Trigger` instead.

File: packages/combobox/src/Combobox/Combobox.tsx [hunk 2]
```diff
@@ -301,9 +288,12 @@ type TriggerProps = React.ComponentProps<typeof TelegraphMenu.Trigger> & {
   size?: TgphComponentProps<typeof TelegraphButton.Root>["size"];
 };
 
-const Trigger = ({ size = "1", ...props }: TriggerProps) => {
+const Trigger = ({ size = "2", ...props }: TriggerProps) => {
```
Comment (kylemcd) [inline packages/combobox/src/Combobox/Combobox.tsx:291]: This default was incorrectly set before when height was being improperly managed.

File: packages/combobox/src/Combobox/Combobox.tsx [hunk 3]
```diff
@@ -301,9 +288,12 @@ type TriggerProps = React.ComponentProps<typeof TelegraphMenu.Trigger> & {
...
   const context = React.useContext(ComboboxContext);
 
+  const minH =
```
Comment (connorlindsey) [inline packages/combobox/src/Combobox/Combobox.tsx:294]: Nit: Could use a map to make this a bit more readable

**Review by connorlindsey — APPROVED** [review]
Looks good!

**Pull request merged**